styleguide-app/Examples/Button.elm

```diff
@@ -11,8 +11,8 @@ import Html.Styled exposing (..)
 import Html.Styled.Attributes exposing (css)
 import ModuleExample as ModuleExample exposing (Category(..), ModuleExample, ModuleMessages)
 import Nri.Ui.AssetPath exposing (Asset)
-import Nri.Ui.Button.V5 as Button
-import Nri.Ui.Icon.V3 as Icon
+import Nri.Ui.Button.V6 as Button
+import Nri.Ui.Icon.V4 as Icon
```

Comment:
<img width="622" alt="screen shot 2018-10-29 at 4 15 55 pm" src="https://user-images.githubusercontent.com/13489089/47685962-f57cb180-db95-11e8-9bd2-7e105fe18e2b.png">

I think these little label-y thingies should be updated too!